### PR TITLE
fixed indentation, naming conventions and the sorting bug

### PR DESCRIPTION
**Pull request by Saatvik-droid**
Fixed the 4 spaces indent convention, a few definition naming violations and the sorting bug

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```
Comment (sayand0122) [inline .idea/vcs.xml:6]: What are this files for?

File: .idea/modules.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/Data-Structures-Algorithms-DSA-.iml" filepath="$PROJECT_DIR$/.idea/Data-Structures-Algorithms-DSA-.iml" />
+    </modules>
+  </component>
+</project>
```
Comment (sayand0122) [inline .idea/modules.xml:8]: And this also?

File: .idea/inspectionProfiles/profiles_settings.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <settings>
+    <option name="USE_PROJECT_PROFILE" value="false" />
+    <version value="1.0" />
+  </settings>
+</component>
```
Comment (sayand0122) [inline .idea/inspectionProfiles/profiles_settings.xml:6]: This?

File: .idea/inspectionProfiles/Project_Default.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    <inspection_tool class="PyTypeCheckerInspection" enabled="false" level="WARNING" enabled_by_default="false" />
+    <inspection_tool class="PyUnresolvedReferencesInspection" enabled="true" level="WARNING" enabled_by_default="true">
+      <option name="ignoredIdentifiers">
+        <list>
+          <option value="cogs.jellyspawn.*" />
+        </list>
+      </option>
+    </inspection_tool>
+  </profile>
+</component>
```
Comment (sayand0122) [inline .idea/inspectionProfiles/Project_Default.xml:13]: This?

File: .idea/Data-Structures-Algorithms-DSA-.iml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="PYTHON_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (sayand0122) [inline .idea/Data-Structures-Algorithms-DSA-.iml:8]: And this? actually  have never worked with xml files so no idea...

Comment (Saatvik-droid) [reply .idea/Data-Structures-Algorithms-DSA-.iml:8]: To be honest I have no idea. I just worked on those python files and just committed them to my fork.

Comment (Saatvik-droid) [reply .idea/Data-Structures-Algorithms-DSA-.iml:8]: Don't merge them or add them to gitignore. Maybe it's a pycharm thing.

Comment (Saatvik-droid) [reply .idea/Data-Structures-Algorithms-DSA-.iml:8]: Yeah I looked it up it's a pycharm thing oof.

Comment (sayand0122) [reply .idea/Data-Structures-Algorithms-DSA-.iml:8]: ok got it...

**Review by sayand0122 — APPROVED** [review]
Great job @Saatvik-droid

**Pull request merged**